themes/jquery/header.php

```diff
@@ -79,11 +79,11 @@
 					</li>
 					<li class="dropdown"><a href="https://jquery.org/support/">Support</a>
 						<ul>
-							<li><a href="http://learn.jquery.com/">Learning Center</a></li>
-							<li><a href="http://try.jquery.com/">Try jQuery</a></li>
-							<li><a href="http://irc.jquery.org/">IRC/Chat</a></li>
-							<li><a href="http://forum.jquery.com/">Forums</a></li>
-							<li><a href="http://stackoverflow.com/tags/jquery/info">Stack Overflow</a></li>
+							<li><a href="https://learn.jquery.com/">Learning Center</a></li>
+							<li><a href="https://try.jquery.com/">Try jQuery</a></li>
```

Comment:
This page is not available via HTTPS. Can you make sure all other links work?

Reply:
thanks for spotting that. They are all good now. 👍